src/utils.ts

```diff
@@ -1,66 +1,29 @@
+import { Common } from "fagc-api-types"
 import { NoAuthError, RequestConfig } from "."
-import { FetchRequestTypes } from "./types/privatetypes"
+import BaseManager from "./managers/BaseManager"
 
 /**
  * A decorator that checks whether an api key is set in the reqConfig object or on the class, if it isnt it throws an error.
  * Sets the `reqConfig.apikey` to be the value of the api key from the original `reqConfig` or the class.
  * 
- * Throws an error if the api key is not set.
+ * @throws NoAuthError if the API key is not set
+ * @returns String to use for the Authorization header
```

Comment:
You updated the functions but didn't update their doc strings which are now wrong.

Reply:
Oops, forgot to change the first part of it. Thanks